Add hybrid identifier (#661)
MewsSystems/gitbook-connector-api: operations/_objects.md

| `NetValue` | number | required | The net value that the tax is calculated from. |
| `TaxValue` | number | required | The value of the tax. |

### Hybrid identifier

> Hybrid identifier is a string value with specific format that allows providing other unique identifier than just unique identifier of the entity. Using other identifiers require providing corresponding prefix as part of the value.

Reviewer: For docs annotations I often find ChatGPT version sounding better:
```suggestion
A hybrid identifier is a string with a specific format that allows the use of alternative unique identifiers, in addition to the entity's unique identifier. When using these alternative identifiers, a corresponding prefix must be included as part of the value.
```

Author: I trier Copilot but that wasn't helpful much 🤣

| Identifier | Prefix | Value example |
| :-- | :-- | :-- | :-- |

> | Unique identifier | - | `a01bc7c3-cfa2-4ad6-a360-5cb8b4004ab5` |

Reviewer: External identifiers should often (if not always) be unique too.
```suggestion
| Primary identifier | - | `a01bc7c3-cfa2-4ad6-a360-5cb8b4004ab5` |
```

> | Primary identifier | - | `a01bc7c3-cfa2-4ad6-a360-5cb8b4004ab5` |

Reviewer 2: nitpick: having `-` as Prefix is misleading, consider keeping the field empty or put something like `(none)` there.

> | External identifier | eid | `eid:COM-123` |

Reviewer 2: nitpick: I'd suggest colon to be treated as part of the prefix.

```suggestion
| External identifier | `eid:` | `eid:COM-123` |
```

| `NetValue` | number | required | The net value that the tax is calculated from. |
| `TaxValue` | number | required | The value of the tax. |

### Hybrid identifier

A hybrid identifier is a string with a specific format that allows the use of alternative unique identifiers, in addition to the entity's unique identifier. When using these alternative identifiers, a corresponding prefix must be included as part of the value.

| Identifier | Prefix | Value example | Description |
| :-- | :-- | :-- | :-- |

> | Primary identifier |  | `a01bc7c3-cfa2-4ad6-a360-5cb8b4004ab5` | Primary identifier of the entity e.g. `Id` field on `Service`.|

Reviewer 3: Just a suggestion:
```suggestion
| Primary identifier | (no prefix) | `a01bc7c3-cfa2-4ad6-a360-5cb8b4004ab5` | Primary identifier of the entity e.g. `Id` field on `Service`.|
```
